idseq_dag/steps/generate_host_genome.py

```diff
@@ -49,6 +47,7 @@ def run(self):
                                          auto_unzip=True)
 
         host_name = self.additional_attributes["host_name"]
+        max_star_part_size = self.additional_attributes.get("max_star_part_size")
```

Comment:
should we keep a default for this?

Reply:
The default is no maximum since we want to move away from sharding.